ibis/backends/postgres/tests/test_geospatial.py

```diff
@@ -192,7 +192,7 @@ def test_geo_spatial_unops(geotable, expr_fn, expected):
             marks=pytest.mark.notimpl(["postgres"]),
         ),
         param(
-            lambda t: t.geo_polygon.contains(ibis.geo_point(30, 10)),
+            lambda t: t.geo_polygon.contains(ibis.literal(30).point(10)),
```

Comment:
We many want to add these constructor-like functions as proper APIs at some point, but that can be done later.